setup.py

```diff
@@ -545,18 +598,24 @@ def run(self):
             "build/pip_data_bin_init.py.in",
             os.path.join(bin_dir, "__init__.py"),
         )
+        # Share the cmake-out location with _BaseExtension.
+        self.cmake_cache_dir = cmake_cache_dir
 
         # Finally, run the underlying subcommands like build_py, build_ext.
         build.run(self)
 
 
 def get_ext_modules() -> List[Extension]:
     """Returns the set of extension modules to build."""
-
     ext_modules = []
     if ShouldBuild.flatc():
         ext_modules.append(
-            BuiltFile("third-party/flatbuffers/flatc", "executorch/data/bin/")
+            BuiltFile(
+                "third-party/flatbuffers/%BUILD_TYPE%/",
```

Comment:
Will `%BUILD_TYPE%` always be the final component of the directory path? If so, we could remove the `%BUILD_TYPE%` placeholder and just add it to the source path inside BuiltFile.

It's ok as-is for now, though.

Reply:
It is, but it does not have access to the installer in the constructor of BuiltFile.
If I understand correctly, obtaining the build configuration requires access to installer.debug.